corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -873,6 +875,21 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
             this.labelWidth = constants.LABEL_WIDTH;
             this.questionTileWidth = constants.FULL_WIDTH;
         }
+
+        function setCustomInputWidth() {
+            const shortWidth = new RegExp(constants.SHORT_WIDTH);
+            const mediumWidth = new RegExp(constants.MEDIUM_WIDTH);
+            $(document).ready(function () {
+                const entryWidgetContainer = $(`#${self.entry.entryId}`).closest('.widget-container');
+                if (self.stylesContains(shortWidth)) {
+                    entryWidgetContainer.removeClass(constants.CONTROL_WIDTH);
```

Comment:
Would this work with question tiles? I think with question tiles (i.e. `3-per-row` appearance attribute for the question), `.widget-container` would have the class `constants.FULL_WIDTH`, not `constants.CONTROL_WIDTH`

Reply:
Good point about question tiles. This should still work the same. I'm removing the `CONTROL_WIDTH` class for the sake of cleanliness. Because this function waits for the page to load and then adds the class pertaining to the appearance attribute it overrides the class that was there already. So if we keep this approach I'd also want to remove the `FULL_WIDTH` class so that there aren't unused classes hanging around, which would be confusing for devs in the future.